### ISSUE TITLE
JAWS can't read collapse Accordion after expand

### ISSUE THREAD

**Issue by jacklotusho** [carbon-design-system/carbon#281]
JAWS can't read collapse Accordion after expand, expecting when accordion expands, the detail information should be read by screen reader.

Test URL http://carbondesignsystem.com/components/accordion/live

**Comment by tw15egan** [comment]
Fixed via https://github.com/carbon-design-system/carbon-components/pull/262
